app/controllers/application_controller/tenancy.rb

```diff
@@ -9,11 +9,10 @@ module ApplicationController::Tenancy
   end
 
   def current_tenant
-    # current_user.try(:current_group).try(:tenant) || Tenant.default_tenant
     @current_tenant ||=
      #  Tenant.where(:subdomain => request.subdomains.last).first ||
      #  Tenant.where(:domain => request.domain).first ||
-      Tenant.default_tenant
+      current_user.try(:tenant) || Tenant.default_tenant
```

Comment:
Maybe it's badly named but it's User#current_tenant which is delegated through current_group.

`current_user.try(:current_tenant) || Tenant.default_tenant`

See: https://github.com/ManageIQ/manageiq/blob/55330e8e2b665b636b420a6192b5cb4e27b23426/app/models/user.rb#L26